docs/software/databases.md

```diff
@@ -39,6 +39,17 @@ Source: [MariaDB](https://mariadb.com/), [LGPL](https://commons.wikimedia.org/w/
     - Username = `root`
     - Password = The same as your root login password, default is `dietpi`
 
+=== "Configuration"
+
+    MariaDB configuration file could be found at: `/etc/mysql/my.cnf`.
+
+    By default, DietPi sets MariaDB configuration to use Unix socket connections. This is a fast way to enable local connectivity, comparing with TCP connections[^2].
+    
+    ```
+    [client-server]
+    socket = /run/mysqld/mysqld.sock
+    ```
+
```

Comment:
This is not the configuration file which should be changed. For adding/overriding settings, instead an override configuration like `/etc/mysql/mariadb.conf.d/99-custom.cnf` should be used. Such is the only one which is assured to be fully respected, all the others are legacy or fallback configs, overwritten by others and those in `/etc/mysql/mariadb.conf.d/` with highest priority. A little mess since for some reason the package still ships all configuration files used by MySQL and additionally a set of new ones only used by MariaDB.

Reply:
I will remove this addition, and let for a short while the task not resolved, to run more tests on this configuration. Thank you for all the details